modules/serverless-python-template.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-developing-python-functions.adoc
+
+[id="serverless-python-template_{context}"]
+= Python function template structure
+
```

Comment:
Just suggestion: one thing I believe is nice to have in the doc is the command to create a python function (`kn func create -l python`).

Reply:
I will be adding the `kn func create` docs at another time, they need their own module 😉 this is just about what's contained in a Python function, not the `kn` tasks for it really.